app/views/search/results.haml

```diff
@@ -29,18 +29,16 @@
     - if @results_total.to_i >= 1
       %nav.navigation--number{ aria:{ label: 'Pagination' } }
         - unless current_page == 1
-          %a.navigation-control--left{ href: "/search?q=#{@query_parameter}&start_index=#{start_index(previous_page)}&count=#{@count}" }
-            =t('pagination.previous')
+          = link_to(t('pagination.previous'), "/search?q=#{@query_parameter}&start_index=#{start_index(previous_page)}&count=#{@count}".html_safe, { class: "navigation-control--left" })
```

Comment:
Could you replace the fixed path '/search' with the routes helper **search_path**